Sezai/index.js

```diff
@@ -0,0 +1,81 @@
+//1. 
+
+let board = [
+  [1, 2, 3],
+  ["quick", "brown", "fox", "jumped", "over", "lazy", "dog"],
+  [true, false]
+];
+
+const items =[]
+
+function getItemsInside(arr){
+for (let valueI of arr){
+    for (let valueJ of valueI){
+    items.push(valueJ)
+}
+}
+}
+getItemsInside(board)
+console.log("items ==> ", items);
+
+/**
+ * items ==>  [
+  1,        2,
+  3,        'quick',
+  'brown',  'fox',
+  'jumped', 'over',
+  'lazy',   'dog',
+  true,     false
+]
+ */
+
+
+ //2.1 & 2.2 & 2.4
+ const doggo = {
+     name: "Sansa",
+     breed: "Rottweiler",
+     favoriteFood: ["steak", "beef", "chickenfilet","humans dinner"]
+    }
```

Comment:
Task 2.4 ==> The result is correct and you have access to favoriteFood, You can also declare a method within doggo object like for example 
favFood() {
    this.favoriteFood.map((food) => console.log("FavFood==>",food));
}
And then after doggo object you can call the function and print the favoriteFood using doggo.favFood()